src/components/pages/Home/RenderHomePage.js

```diff
@@ -1,34 +1,46 @@
-import React from 'react';
+import React, { useEffect } from 'react';
```

Comment:
Very clean looking formatting. Its easy to understand what this code is doing.

Reply:
looks good!